modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/RuntimeProvisioningService.java

```diff
@@ -23,45 +23,177 @@
 
 public interface RuntimeProvisioningService {
```

Comment:
This is fixed in following pull request.
https://github.com/wso2/product-af/pull/800/files#diff-350fba7594b78452a912ac7c6e35f0a2R23